packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseWalletConfirmationStep1.tsx

```diff
@@ -0,0 +1,80 @@
+import { Rows, Card, Columns, Button, Text } from '@trezor/components';
+import { HELP_CENTER_PASSPHRASE_URL } from '@trezor/urls';
+import { Translation } from 'src/components/suite/Translation';
+import { TrezorLink } from 'src/components/suite/TrezorLink';
+import { PassphraseHeading } from './PassphraseHeading';
+import { ContentType } from './types';
+
+type PassphraseWalletConfirmationStep1Props = {
+    setContentType: React.Dispatch<React.SetStateAction<ContentType>>;
```

Comment:
I am kinda in favor of not using `React.` but to import it. Is this accidental or you think using `React.` is better?

Reply:
Actually I think it doesnt matter at all, both ways are acceptable for me and I think we don't have to define strict line here. I just copy paste it from somewhere and add import automatically.